### DISCUSSION TITLE
Playstyle like in this Vid: https://youtu.be/C97PvIkpVag

### DISCUSSION THREAD

**Discussion by kleines-L** [ppy/osu#14452] [Ideas]
Video link: 

https://youtu.be/C97PvIkpVag

**Comment by mk56-spn** [comment]
this is not how you make an idea you need to explain what you mean otherwise people will not understand what it is you want, are you saying there should be support for having a certain region of the screen for tapping and another for a stylus / finger  dragging ?

**Reply by kleines-L** [reply]
not quite what I mean. So if you hold your finger in one place and tap on another random place, then the "tap" happens where the first finger is

**Comment by peppy** [comment]
This is being tracked at https://github.com/ppy/osu/issues/4201

**Reply by kleines-L** [reply]
when would this feature come in the game?

**Reply by peppy** [reply]
when it's coded and reviewed.
